sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/TraceHelperTests.cs

```diff
@@ -226,8 +229,9 @@ public void ActivityWithExceptionEventCreatesExceptionTelemetry()
             var exceptionMessage = "Exception Message";
             using ActivitySource activitySource = new ActivitySource(ActivitySourceName);
             using var activity = activitySource.StartActivity(
-               ActivityName,
-               ActivityKind.Server);
+                ActivityName,
```

Comment:
leftover from clean up?

Reply:
was just an odd whitespace. i fixed it here since i was already working in the class.